Makefile

```diff
@@ -15,6 +15,7 @@
 .PHONY: get-realsense-serial-num
 
 MKDOCS_IMAGE ?= asc-mkdocs
+ARCHITECTURE ?= arc  # arc|flex|max
```

Comment:
what does max mean?
I would suggest changing ARCHITECTURE to `GPU_TYPE` since architecture is too generic, it means something very different in OS and thus could be misleading